data_source_obmcs_core_instance_test.go

```diff
@@ -0,0 +1,129 @@
+// Copyright (c) 2017, Oracle and/or its affiliates. All rights reserved.
+
+package main
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	baremetal "github.com/oracle/bmcs-go-sdk"
+
+	"github.com/stretchr/testify/suite"
+	"fmt"
+	"time"
+)
+
+type DatasourceCoreInstanceTestSuite struct {
+	suite.Suite
+	Client 			*baremetal.Client
+	Config			string
+	Provider		terraform.ResourceProvider
+	Providers 		map[string]terraform.ResourceProvider
+	ResourceName	string
+}
+
+
+var instance_display_name string
+var metadata_ssh_key_entry = "ssh_authorized_keys"
```

Comment:
I'm not seeing what this variable is for, just use "ssh_authorized_keys" in your assertion.